src/i18n/en/translation.en.json

```diff
@@ -1218,6 +1218,7 @@
     "loading-flow": "Loading flow",
     "loading-error": "Loading error",
     "messages": {
+      "successfully-recovered-password": "You successfully recovered your account. Please change your password within the next 15 minutes.",
```

Comment:
Is the message not more accurate as:
"Please enter a new password within the next 15 minutes to complete your password reset."

Reply:
Sure, I'll put this. It was like this in the task, just that, but it's just a minute I'll change it.